examples/csv-io/csv_io.bal

```diff
@@ -61,7 +61,7 @@ public function main() returns @tainted error? {
         }
     } else {
         log:printError("An error occurred while creating table: ",
-                        err = tblResult);
+                        err = <error>tblResult);
     }
```

Comment:
I think instead of casting to error, it would be better if we change this to 

```ballerina
    if (tblResult is table<record {}>) {
        table<Employee> empTable = <table<Employee>> tblResult;
        foreach var rec in empTable {
            io:println(rec);
        }
    } else {
        log:printError("An error occurred while creating table: ",
                        err = tblResult);
    }
```

or even

```ballerina
    if (tblResult is table<record {}>) {
        foreach var rec in tblResult {
            io:println(rec);
        }
    } else {
        log:printError("An error occurred while creating table: ",
                        err = tblResult);
    }
```

@anupama-pathirage WDYT?

Reply:
+1 for the first option.  Otherwise it looks like no point in passing `Employee` to the  `getTable` function in `var tblResult = rCsvChannel2.getTable(Employee);`